packages/travix-ui-kit/components/badge/badge.js

```diff
@@ -103,6 +105,10 @@ Badge.propTypes = {
    * The Badge's position
    */
   position: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
+  /**
+   * The Badge's size
+   */
+  size: PropTypes.oneOf(['default', 'small']),
```

Comment:
What do you think about standard enum for sizes? Like we have planned for button, for example

Reply:
could you please explain more about it ?
Sorry, I did not participate in the discussion

Reply:
@rbardini explained below, would be nice to follow the standard we have for sizes `s`, `m`, `l` etc